sparse_strips/vello_example_scenes/src/path.rs

```diff
@@ -0,0 +1,548 @@
+        y = 0.0;
+        for start in cap_styles {
+            for end in cap_styles {
+                let stroke = Stroke::new(20.0)
+                    .with_start_cap(start)
+                    .with_end_cap(end)
+                    .with_dashes(0.0, [10.0, 21.0]);
+                ctx.set_transform(
+                    root_transform * Affine::translate((0.0, y + 30.0)) * t * self.transform,
+                );
+                ctx.set_paint(colors[color_idx]);
+                ctx.set_stroke(stroke);
+                ctx.stroke_path(&simple_stroke);
+                y += Y_OFFSET;
+                color_idx = (color_idx + 1) % colors.len();
+            }
+        }
+
+        // Cap and join combinations
+        let t = Affine::translate((X_OFFSET, 0.0)) * t;
+        y = 0.0;
+        for cap in cap_styles {
+            for join in join_styles {
+                let stroke = Stroke::new(20.0).with_caps(cap).with_join(join);
+                ctx.set_transform(
+                    root_transform * Affine::translate((0.0, y + 30.0)) * t * self.transform,
+                );
+                ctx.set_paint(colors[color_idx]);
+                ctx.set_stroke(stroke);
+                ctx.stroke_path(&join_stroke);
+                y += Y_OFFSET;
+                color_idx = (color_idx + 1) % colors.len();
+            }
+        }
+
+        // Miter limit
+        let t = Affine::translate((X_OFFSET, 0.0)) * t;
+        y = 0.0;
+        for ml in miter_limits {
+            let stroke = Stroke::new(10.0)
+                .with_caps(Cap::Butt)
+                .with_join(Join::Miter)
+                .with_miter_limit(ml);
+            ctx.set_transform(
+                root_transform * Affine::translate((0.0, y + 30.0)) * t * self.transform,
+            );
+            ctx.set_paint(colors[color_idx]);
+            ctx.set_stroke(stroke);
+            ctx.stroke_path(&miter_stroke);
+            y += Y_OFFSET;
+            color_idx = (color_idx + 1) % colors.len();
+        }
+
+        // Closed paths
+        for (i, join) in join_styles.iter().enumerate() {
+            // The cap style is not important since a closed path shouldn't have any caps.
+            let stroke = Stroke::new(10.0)
+                .with_caps(cap_styles[i])
+                .with_join(*join)
+                .with_miter_limit(5.0);
+            ctx.set_transform(
+                root_transform * Affine::translate((0.0, y + 30.0)) * t * self.transform,
+            );
+            ctx.set_paint(colors[color_idx]);
+            ctx.set_stroke(stroke);
+            ctx.stroke_path(&closed_strokes);
+            y += Y_OFFSET;
+            color_idx = (color_idx + 1) % colors.len();
+        }
+    }
+}
+
+/// Funky paths scene state
+#[derive(Debug, Default)]
+pub struct FunkyPathsScene {}
+
+impl FunkyPathsScene {
+    /// Create a new funky paths scene
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl ExampleScene for FunkyPathsScene {
```

Comment:
I noticed that `FunkyPathsScene` example is rendering incorrectly. cc @LaurenzV


<img width="210" height="241" alt="image" src="https://github.com/user-attachments/assets/28a7ceea-87e8-435d-bcbc-07be70b80513" />

Reply:
Filed issue #1240